layers/core_checks/cc_ray_tracing.cpp

```diff
@@ -2048,11 +2048,11 @@ bool CoreChecks::PreCallValidateGetRayTracingShaderGroupStackSizeKHR(VkDevice de
                          string_VkPipelineBindPoint(pipeline_state->pipeline_type));
     } else {
         const auto &create_info = pipeline_state->RayTracingCreateInfo();
-        if (group >= create_info.groupCount) {
-            skip |=
-                LogError("VUID-vkGetRayTracingShaderGroupStackSizeKHR-group-03608", pipeline, error_obj.location.dot(Field::group),
-                         "(%" PRIu32 ") must be less than the number of shader groups in pipeline (%" PRIu32 ").", group,
-                         create_info.groupCount);
+        const uint32_t total_group_count = CalcTotalShaderGroupCount(*pipeline_state);
+        if (group >= total_group_count) {
+            skip |= LogError(
+                "VUID-vkGetRayTracingShaderGroupStackSizeKHR-group-03608", pipeline, error_obj.location.dot(Field::group),
+                "(%" PRIu32 ") must be less than the number of shader groups in pipeline (%" PRIu32 ").", group, total_group_count);
```

Comment:
```suggestion
                "(%" PRIu32 ") must be less than the number of shader groups in pipeline (%" PRIu32 "). (groupCount has %" PRIu32 " and %" PRIu32 " are from pipeline libraries)", group, total_group_count, create_info.groupCount, (total_group_count - create_info.groupCount));                               
```

I could see someone going "WTF, my groupCount is only X, VVL bug!"